tests/functional/utils/test_abi.py

```diff
@@ -44,5 +44,4 @@ def test_parse_type(s):
     #   - ,int
     #   - int,
     # See tests in `tests_contracts` for specific ABI parsing tests.
-
-    assert parse_type(s)
```

Comment:
Why was this removed? Does it fail now? Why does it fail?

Reply:
because:

- `parse_type` has a different input type
- could not set up a local test environment (precise issue in the description of this PR)

If someone can help me out with the exact steps to set up the test environment, I will address this as well.

Reply:
I am not sure about your environment issue, I have not seen that one before.
Seems like `pygit2` is expecting some `libssl` to be installed but it can't find it.
I'd like for us to not rely on `pygit2` in the future... but for now I would figure out how to get that working and document when you had to do.

Reply:
I tried solving it by installing a specific libssl version on my machine. That didn't help. So I will create a blank repo and write the tests in there and just move it (copy/paste) to this repo (since those tests do not rely on any lib code other than `parse_type`.

Reply:
Ok, the only places it is failing now is these three:

```
FAILED tests/functional/test_contract_instance.py::test_solidity_structs_with_array - assert False
FAILED tests/functional/test_contract_instance.py::test_vyper_struct_arrays - AssertionError: assert False
FAILED tests/functional/test_contract_instance.py::test_solidity_static_struct_arrays - assert False
```

and the changes in this PR certainly do not cause addresses to not be returned in checksum format. I bet the current main has these tests failing too.